dotnet/src/Functions/Functions.OpenApi/RestApiOperationRunner.cs

```diff
@@ -163,6 +164,7 @@ internal sealed class RestApiOperationRunner
         requestMessage.Headers.Add("User-Agent", !string.IsNullOrWhiteSpace(this._userAgent)
             ? this._userAgent
             : HttpHeaderValues.UserAgent);
+        requestMessage.Headers.Add(HttpHeaderNames.SemanticKernelVersion, typeof(RestApiOperationRunner).Assembly.GetName().Version.ToString());
```

Comment:
nit: we may think about having a reusable method that would return assembly version HttpHeaderValues.GetAssemblyVersion(typeof(RestApiOperationRunner).Assembly)